test/runner.test.js

```diff
@@ -45,9 +45,19 @@ describe("npx create-snowpack-app", () => {
     it(`--template @snowpack/${template}`, async () => {
       const cwd = path.join(TEMPLATES_DIR, template);
 
-      // run yarn build
+      // pre-build: minify output
+      const snowpackConfig = path.join(cwd, "snowpack.config.json");
+      const originalConfig = fs.readFileSync(snowpackConfig, "utf8");
+      const config = JSON.parse(originalConfig);
+      config.buildOptions = { ...(config.buildOptions || {}), minify: false };
+      fs.writeFileSync(snowpackConfig, JSON.stringify(config), "utf8");
```

Comment:
The reason for this weirdness is that we’re testing user-facing configs, so we absolutely don’t want to have anything in there meant for testing only, and not users. So we temporarily modify the config while it’s running, then restore the original. It’s a bit hacky, but it does result in a more robust test while keeping the templates user-focused.